microsoft-365/admin/add-users/reset-passwords.md

```diff
@@ -130,32 +130,39 @@ These steps work for a business with tens of users. If you have hundreds or thou
 
 5. Enter an email address where you can receive the temporary passwords. You'll need to notify your users what their temporary passwords are.
   
-## Reset business passwords in bulk
+## Password reset at scale
+
+For complete information on how to change passwords at scale or enforce password change at next login via Microsoft Graph Powershell check out this blog post by Padure Sergio: [Force password change for all users in Microsoft 365 - Microsoft Graph Powershell Edition](https://blog.raindrops.dev/blog/force-password-change-for-all-users-in-office-365/)
```

Comment:
We do not reference third-party sites and blogs in Microsoft documentation, therefore this reference should not be included.

Reply:
Uh, is that a new policy? A lot of docs have links to blogs for a more extensive explanation of a certain subject or additional code